bin/build_pipeline_heka.sh

```diff
@@ -52,6 +58,39 @@ fi
 
 source build.sh
 
+echo 'Installing lua-geoip libs'
+cd $BASE/build
+if [ ! -d lua-geoip ]; then
+    # Fetch the lua geoip lib
+    git clone https://github.com/agladysh/lua-geoip.git
+    cd lua-geoip
+    # from 'make.sh'
+    gcc -O2 -fPIC -I${LUA_INCLUDE_PATH} -c src/*.c -Isrc/ -Wall --pedantic -Werror --std=c99 -fms-extensions
+
+    UNAME=$(uname)
+    case $UNAME in
+    Darwin)
+        echo "Looks like OSX"
+        gcc -bundle -undefined dynamic_lookup database.o city.o -o city.so
```

Comment:
Just use the case to set the flags and have one set of gcc commands gcc $(SO_FLAGS) database.o city.o -o city.so ect